setup/pi/setup-teslausb

```diff
@@ -921,6 +921,13 @@ make_root_fs_readonly
 
 upgrade_packages
 
+if [ -n "${INSTALL_USER_REQUESTED_PACKAGES:-}" ]
+then
+  setup_progress "Installing user requested packages: ${INSTALL_USER_REQUESTED_PACKAGES}"
+  # shellcheck disable=SC2086
```

Comment:
I think this "shellcheck disable" can can be avoided by using `eval "DEBIAN_FRONTEND=noninteractive apt-get -y --force-yes install $INSTALL_USER_REQUESTED_PACKAGES"`, which isn't pretty but I like it slightly better than disabling shellcheck checks.

Reply:
a nicer way would be to expand the setup variable into an array (or make it an array to begin with) and then expanding the array.
Something like:
```
read -r -a PACKAGES <<< "${INSTALL_USER_REQUESTED_PACKAGES[@]}"
DEBIAN_FRONTEND=noninteractive apt-get -y --force-yes install "${PACKAGES[@]}"
```
should work for both both a simple space-separated list as well as a proper array.

Reply:
Thank you @marcone.